### PR TITLE
integrate kappa-core, remove browser support for now

### PR DESCRIPTION
**Pull request by okdistribute**
I removed browser support directly for now since that's likely just more features to support. 

This should be a major bump, so that any older modules that depend upon osm-p2p can stay on the older stack.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,58 +1,45 @@
...
+var Deflevel = require('deferred-leveldown')
+var raf = require('random-access-file')
 
 module.exports = function (opts) {
   if (typeof opts === 'string') opts = { dir: opts }
   if (!opts) opts = {}
-  var dir = opts.dir || path.resolve('./osm-p2p-data')
-  var defIndexDB = new deflevel()
-  var defLogDB = new deflevel()
+  var dir = opts.dir || path.resolve('./osm-p2p-kappa')
```
Comment (okdistribute) [inline index.js:14]: changed the name to not collide with other databases that could exist, what do you think @noffle ?

File: index.js [hunk 2]
```diff
@@ -1,58 +1,45 @@
...
   })
-  var db = levelup(defIndexDB)
+
   var osm = osmdb({
-    log: log,
-    db: db,
-    store: defStore
+    core: kappa(dir, {valueEncoding: 'json'}),
+    index: index,
+    storage: raf
```
Comment (okdistribute) [inline index.js:42]: is this the `random-access-*` module that we want to use?

Comment (hackergrrl) [reply index.js:42]: yep! that's the canonical on-disk module, which I think maps well onto the old on-disk level storage

File: index.js (hunk 1 above)
Comment (hackergrrl) [reply index.js:14]: I'm down with this :+1:

**Review by hackergrrl — APPROVED** [review]

**Comment by hackergrrl** [comment]
3.0.0

**Pull request merged**